java-surefire/src/main/java/org/sonar/plugins/surefire/SurefireJavaParser.java

```diff
@@ -49,7 +50,7 @@
  */
 public class SurefireJavaParser implements BatchExtension {
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Remove this use of "BatchExtension"; it is deprecated. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://dory.sonarsource.com/sonarqube/coding_rules#rule_key=squid%3ACallToDeprecatedMethod)